R/teal_slices-store.R

```diff
@@ -27,11 +32,14 @@
 #'   teal.slice::teal_slice(dataname = "data", expr = "x > 0", id = "positive_x", title = "Positive x")
```

Comment:
Since `teal` `Depends` on `teal.slice`, we don't need prefixes in these examples. We used to leave them in on purpose in order to emphasise the external nature of some functions but now that we have been removing them, consider removing this one as well. The example will work even without `library(teal.slice)`.

```suggestion
#'   teal_slice(dataname = "data", expr = "x > 0", id = "positive_x", title = "Positive x")
```